src/widget/form/genericchatform.cpp

```diff
@@ -199,6 +203,7 @@ GenericChatForm::GenericChatForm(QWidget *parent)
             this, &GenericChatForm::onChatContextMenuRequested);
 
     new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_L, this, SLOT(clearChatArea()));
+    new QShortcut(Qt::ALT + Qt::Key_Q, this, SLOT(quoteSelectedText()));
```

Comment:
`CTRL` + `SHIFT` + `v` is a common shortcut to paste unformatted text in different software. I think it is a bad way to use it for another purposes.